site/public/js/drag-and-drop.js

```diff
@@ -1026,6 +1061,13 @@ function handleUploadCourseMaterials(csrf_token, expand_zip, cmPath, requested_p
     $.ajax({
```

Comment:
HandleCourse materials isn't actually part of student submission and is actually only used by instructors.
Since the interface is different they won't see the progress bar on that page.

For this PR I think its best to remove this and consider adding loading bars for modals in a different PR/issue

Reply:
sure